cmd/ocm/edit/machinepool/cmd.go

```diff
@@ -61,6 +66,27 @@ func init() {
 		"Restrict application route to direct, private connectivity.",
 	)
 
+	flags.BoolVar(
+		&args.autoscalingEnabled,
+		"enable-autoscaling",
+		false,
+		"Enable autoscaling for the machine pool.",
+	)
+
+	flags.IntVar(
+		&args.minReplicas,
+		"min-replicas",
+		0,
+		"Minimum number of machines for the machine pool.",
+	)
+
+	flags.IntVar(
+		&args.maxReplicas,
+		"max-replicas",
+		0,
+		"Maximum number of machines for the machine pool.",
+	)
```

Comment:
Can you use `arguments.AddAutoscalingFlags(...)` here?

Reply:
I can.
That would require changing the implementation to be like the cluster - Autoscaling struct.
I should probably do the same for `create`